schema/CodeError.schema.yaml

```diff
@@ -6,9 +6,8 @@ status: unstable
 category: code
 description: An error that occured when parsing, compiling or executing some Code.
 properties:
-  kind:
-    # not called `type` because this is a conflict or not supported by Python
-    '@id': stencila:errorKind
+  errorType:
+    '@id': stencila:errorType
     description: The type of error.
     type: string
```

Comment:
Aside for consumer app usability: What do you think about constraining `type` to be `OneOf` severity levels, say from Logga?

https://github.com/stencila/logga/blob/17463771735a4fb7f427bb2348057c201198650a/index.ts#L77-L85

Reply:
I've added a comment on the intended purpose of this property: https://github.com/stencila/schema/pull/161/commits/a6ad777252c8d3216633096ef0b2b68b86de173a#diff-61aa5767b327ea79bfde73709a862d70R13 Which hopefully clarifies why I don't think this is appropriate. We could add another property using an `enum` that apps e.g. Pyla, Rasta could translate language specific errors to e.g. `ZeroDivisionError` -> `fatal`. But I think that is something for another time, if at all.

Reply:
thanks for the clarification. To give some context for why I thought it'd be useful, was because of [usage in Designa](https://github.com/stencila/designa/tree/master/packages/components/src/components/error#properties). We fall back to showing a `warning` error type, but having some kind of agreed upon and finite number of displayable error types would be desirable for properly handling error messages.